libs/spice/Makefile

```diff
@@ -32,6 +32,30 @@ define Package/libspice-server
   DEPENDS:=+glib2 +libjpeg +libopenssl +pixman +zlib
 endef
 
+MESON_ARGS += \
+	-Dgstreamer=no \
+	-Dlz4=false \
+	-Dsasl=false \
+	-Dcelt051=disabled \
+	-Dopus=disabled \
```

Comment:
I'd like to at least keep the original comment singling out these two as related "audio codec"

Reply:
Done.